projects/core/components/calendar/calendar.component.ts

```diff
@@ -81,26 +82,22 @@ export class TuiCalendarComponent implements TuiWithOptionalMinMax<TuiDay> {
 
     year: TuiYear | null = null;
 
-    readonly disabledItemHandlerMapper: TuiMapper<
-        TuiBooleanHandler<TuiDay>,
-        TuiBooleanHandler<TuiDay>
-    > = (disabledItemHandler, min: TuiDay, max: TuiDay) => item =>
-        item.dayBefore(min) || item.dayAfter(max) || disabledItemHandler(item);
-
-    get computedMonth(): TuiMonth {
+    ngOnInit(): void {
```

Comment:
Month is an input so it can change. Therefore ngOnInit is not gonna do the trick. I think you can switch this to:

```
@Input('month')
set monthSetter(month: TuiMonth) {
... your clamping of the month to the valid value
}

month = TuiMonth.currentLocal();
```